packages/now-ruby/now_init.rb

```diff
@@ -59,11 +59,19 @@ def webrick_handler(httpMethod, path, body, headers)
 
   server.shutdown
   Thread.kill(th)
+  
+  # Forced string encoding is required due to upstream issue with Net::HTTP
+  # https://bugs.ruby-lang.org/issues/15517
+  res_headers = res.each_capitalized.to_h
+  res_encoding = "UTF-8"
+  if res_headers["Content-Type"] && res_headers["Content-Type"].include?("charset=")
+    res_encoding = res_headers["Content-Type"].match(/charset=([^;]*)/)[1]
+  end
 
   {
     :statusCode => res.code.to_i,
-    :headers => res.each_capitalized.to_h,
-    :body => res.body,
+    :headers => res_headers,
+    :body => res.body.force_encoding(res_encoding),
```

Comment:
I think the think we need to call `force_encoding` as well as `encode`.

This seems to work:

```ruby
res.body.force_encoding('UTF-8')
res.body = res.body.encode('UTF-8')
```

Reply:
Seems like that would work when in the deployment context, yeah.